src/components/ToastContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import { ToastContainer as ToastifyContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { styled } from 'stitches.config';
+
+const StyledContainer = styled(ToastifyContainer, {
+  '.Toastify__toast': {
+    background: '$formInputBackground',
+    borderRadius: '9px',
+    border: '1px solid $formInputBackground',
+    minWidth: '400px',
+    maxWidth: '640px',
+    width: '80%',
+  },
+  '.Toastify__toast-icon': {
+    marginInlineEnd: '20px',
+  },
+  '.Toastify__progress-bar': {
+    height: '1px',
+  },
+  '.Toastify__toast--info': {},
```

Comment:
using no border for `info` ...

Reply:
As discussed, I think we will want a border for the default state as there are some overlapping grey/white problems.